devito/ops/operator.py

```diff
@@ -107,5 +109,7 @@ def hcode(self):
         return ''.join(str(kernel) for kernel in self._ops_kernels)
 
     def _compile(self):
+        self._includes.append('%s.h' % self._soname)
         if self._lib is None:
             self._compiler.prepare_ops(self._soname, str(self.ccode), str(self.hcode))
```

Comment:
is `prepare_ops` really necessary? can we not keep everything in  `jit_copmpile` ?

Reply:
Well noted.  Now we can move this call to the first line of jit_compile, keeping the `def prepare_ops` to have a clean code. 
I think we can rename `prepare_ops`.  Something like `translate_ops`.

Reply:
sounds good to me. Just make it a private method, since it's never going to be used by the outside world

Reply:
Ok. I will change to private.